data/pokedex.js

```diff
@@ -9346,6 +9376,7 @@ let BattlePokedex = {
 		evoLevel: 35,
 		eggGroups: ["Field"],
 		inheritsFrom: "darmanitan",
+		hide: "SpriteMinute",
```

Comment:
Not sure what property you're going here (further elaboration from myself and TDP on Discord)